packages/terra-form-field/src/Field.jsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import IconError from 'terra-icon/lib/icon/IconError';
+import 'terra-base/lib/baseStyles';
+import styles from './Field.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Children the Field contains.
+   */
+  children: PropTypes.node,
+  /**
+   * Error message for when the input is invalid on validation.
+   */
```

Comment:
Nit: This sounds like the Field is doing validation, it might be more direct to say something like `Error message for when isInvalid is true`

Reply:
I would go further and just say `when the input is invalid`